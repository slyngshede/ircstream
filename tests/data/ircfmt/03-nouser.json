{
  "$schema": "/mediawiki/recentchange/1.0.0",
  "meta": {
    "uri": "https://ru.wikipedia.org/wiki/%D0%A0%D0%BE%D1%81%D1%81%D0%B8%D0%B9%D1%81%D0%BA%D0%B0%D1%8F_%D0%BE%D0%BA%D0%BA%D1%83%D0%BF%D0%B0%D1%86%D0%B8%D1%8F_%D0%A5%D0%B0%D1%80%D1%8C%D0%BA%D0%BE%D0%B2%D1%81%D0%BA%D0%BE%D0%B9_%D0%BE%D0%B1%D0%BB%D0%B0%D1%81%D1%82%D0%B8",
    "request_id": "f48bed22-94c4-419b-9deb-1ea0e17369bc",
    "id": "95187f4d-688a-4823-b669-b4f821eeb97e",
    "dt": "2024-10-01T19:17:48Z",
    "domain": "ru.wikipedia.org",
    "stream": "mediawiki.recentchange",
    "topic": "codfw.mediawiki.recentchange",
    "partition": 0,
    "offset": 1135887187
  },
  "id": 494309538,
  "type": "edit",
  "namespace": 0,
  "title": "Российская оккупация Харьковской области",
  "title_url": "https://ru.wikipedia.org/wiki/%D0%A0%D0%BE%D1%81%D1%81%D0%B8%D0%B9%D1%81%D0%BA%D0%B0%D1%8F_%D0%BE%D0%BA%D0%BA%D1%83%D0%BF%D0%B0%D1%86%D0%B8%D1%8F_%D0%A5%D0%B0%D1%80%D1%8C%D0%BA%D0%BE%D0%B2%D1%81%D0%BA%D0%BE%D0%B9_%D0%BE%D0%B1%D0%BB%D0%B0%D1%81%D1%82%D0%B8",
  "comment": "",
  "timestamp": 1727810268,
  "bot": false,
  "notify_url": "https://ru.wikipedia.org/w/index.php?diff=140520812&oldid=140417539",
  "minor": true,
  "length": {
    "old": 159523,
    "new": 159523
  },
  "revision": {
    "old": 140417539,
    "new": 140520812
  },
  "server_url": "https://ru.wikipedia.org",
  "server_name": "ru.wikipedia.org",
  "server_script_path": "/w",
  "wiki": "ruwiki",
  "parsedcomment": ""
}
