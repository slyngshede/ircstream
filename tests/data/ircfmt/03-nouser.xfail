User attribute is missing. It was correctly emitted on irc.wikimedia.org. Likely an EventStreams bug.
