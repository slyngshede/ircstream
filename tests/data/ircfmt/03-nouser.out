#ru.wikipedia
14[[07Российская оккупация Харьковской области14]]4 M10 02https://ru.wikipedia.org/w/index.php?diff=140520812&oldid=140417539 5* 03A lost forgotten sad spirit 5* (+0) 10
